crates/ruff_python_ast/src/all.rs

```diff
@@ -15,17 +16,41 @@ bitflags! {
     }
 }
 
+/// Abstraction for a string inside an `__all__` definition,
+/// e.g. `"foo"` in `__all__ = ["foo"]`
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct DunderAllName<'a> {
+    /// The value of the string inside the `__all__` definition
+    pub name: &'a str,
```

Comment:
Nit: I'd suggest removing the `pub` from these accesses, and add a getter for `name`.